### PR TITLE
Cps/implement 3d incompressible potential element01

### PR DESCRIPTION
**Pull request by inigolcanalejo**
In this PR the 3D element is added.

This is the first PR of a series of upcoming PRs to implement the 3D incompressible potential element. Tentative list of PRs:

1. Preparation: add the 3d incompressible element
2. Add the first version of the define_wake_process_3d.py
3. Add the 3d wake constraint

### CODE REVIEW

File: applications/CompressiblePotentialFlowApplication/compressible_potential_flow_application.h [hunk 1]
```diff
@@ -100,6 +100,7 @@ class KRATOS_API(COMPRESSIBLE_POTENTIAL_FLOW_APPLICATION) KratosCompressiblePote
     ///@{
 
     const IncompressiblePotentialFlowElement<2,3> mIncompressiblePotentialFlowElement2D3N;
+	const IncompressiblePotentialFlowElement<3, 4> mIncompressiblePotentialFlowElement3D4N;
```
Comment (marcnunezc) [inline applications/CompressiblePotentialFlowApplication/compressible_potential_flow_application.h]: the indentation format here is different from the rest

**Review by marcnunezc — COMMENTED** [review]
Nice! Just a minor comment regarding format

File: applications/CompressiblePotentialFlowApplication/compressible_potential_flow_application.h (hunk 1 above)
Comment (inigolcanalejo) [reply applications/CompressiblePotentialFlowApplication/compressible_potential_flow_application.h]: done!

**Review by marcnunezc — DISMISSED** [review]

**Comment by marcnunezc** [comment]
There are conflict with the master after #5246 though!

**Comment by inigolcanalejo** [comment]
Solved the conflict :+1:

File: applications/CompressiblePotentialFlowApplication/custom_utilities/potential_flow_utilities.cpp [hunk 2]
```diff
@@ -214,5 +215,20 @@ template array_1d<double, 2> ComputeVelocityLowerWakeElement<2, 3>(const Element
...
+    const Element& rElement, const array_1d<double, 4>& rDistances);
+template BoundedVector<double, 4> GetPotentialOnUpperWakeElement<3, 4>(
+    const Element& rElement, const array_1d<double, 4>& rDistances);
+template BoundedVector<double, 4> GetPotentialOnLowerWakeElement<3, 4>(
+    const Element& rElement, const array_1d<double, 4>& rDistances);
+template array_1d<double, 3> ComputeVelocityNormalElement<3, 4>(const Element& rElement);
+template array_1d<double, 3> ComputeVelocityUpperWakeElement<3, 4>(const Element& rElement);
+template array_1d<double, 3> ComputeVelocityLowerWakeElement<3, 4>(const Element& rElement);
+template array_1d<double, 3> ComputeVelocity<3, 4>(const Element& rElement);
+template double ComputeIncompressiblePressureCoefficient<3, 4>(const Element& rElement, const ProcessInfo& rCurrentProcessInfo);
```
Comment (marcnunezc) [inline applications/CompressiblePotentialFlowApplication/custom_utilities/potential_flow_utilities.cpp:232]:
```suggestion
template double ComputeIncompressiblePressureCoefficient<3, 4>(const Element& rElement, const ProcessInfo& rCurrentProcessInfo);
template const bool CheckIfElementIsCutByDistance<3, 4>(const BoundedVector<double, 4>& rNodalDistances);
```
we can also add the template of the new Check method for 3D

Comment (inigolcanalejo) [reply applications/CompressiblePotentialFlowApplication/custom_utilities/potential_flow_utilities.cpp:232]: definitely

**Review by marcnunezc — DISMISSED** [review]
OK

**Comment by inigolcanalejo** [comment]
There is another conflict after https://github.com/KratosMultiphysics/Kratos/pull/4686

**Comment by inigolcanalejo** [comment]
Now it should be fine

**Review by marcnunezc — APPROVED** [review]

**Pull request merged**